README.md

```diff
@@ -1,109 +1,88 @@
-# Luxtorpeda-dev
+# 🚋 Luxtorpeda: Enhancing your gaming experience on Linux and Steam Deck
 
 [![Luxtorpeda project Discord](https://img.shields.io/discord/514567252864008206.svg?label=discord)](https://discord.gg/8mFhUPX)
 [![Downloads](https://img.shields.io/github/downloads/luxtorpeda-dev/luxtorpeda/total.svg)](https://github.com/luxtorpeda-dev/luxtorpeda/releases)
 [![Build Action](https://github.com/luxtorpeda-dev/luxtorpeda/actions/workflows/build.yml/badge.svg)](https://github.com/luxtorpeda-dev/luxtorpeda/actions/workflows/build.yml)
 [![GitHub contributors](https://img.shields.io/github/contributors/luxtorpeda-dev/luxtorpeda)](https://github.com/luxtorpeda-dev/luxtorpeda/graphs/contributors)
 [![GitHub Sponsors](https://img.shields.io/github/sponsors/d10sfan)](https://github.com/sponsors/d10sfan)
 
+Luxtorpeda transforms Windows games into native Linux titles, offering improved performance, fewer bugs, and sometimes even enhanced graphics and modern features.
 
-Steam Play compatibility tool to run games using native Linux engines
+## 🚀 Elevate Your Gaming: Luxtorpeda Unleashes Linux Compatibility for Your Favorite Steam Titles
 
-![screenshot-0](https://user-images.githubusercontent.com/3967/61964568-7b674500-afce-11e9-9c42-ef6cc1b425b6.png)
+Are you a Linux or Steam Deck user feeling left out when it comes to enjoying your favorite Steam games? Look no further than Luxtorpeda, your ticket to an enhanced gaming experience on Linux and the Steam Deck. Many Steam games lack native Linux support, or even if they do, there's often a better way to run them. Luxtorpeda steps in as a game-changer, allowing you to run a growing list of titles within a native Linux game engine, be it open source or otherwise.
 
-## Pre-requisites
+## 📖 About
 
-Compatibility tool will work on any modern 64-bit Linux distribution.
-All packaged games are compiled for Steam Runtime Sniper environment and any additional dependencies are provided in the download.
+### 🎨 Features
 
-**Using Luxtorpeda with [Steam native runtime](https://wiki.archlinux.org/index.php/Steam/Troubleshooting#Steam_native_runtime) may or may not work, but is not supported.**
+- 🐧 Modern Native Linux engines
+- :godmode: Better performance and less bugs
+- 🥇 Modern features
+- 🌟 Improved graphics
+- 🐤 Easy to use with automatic installation and configuration
+- 📖 Free and Open Source
+- 👨‍👨‍👧‍👦 Community built and supported
 
-## Installation (using tarball)
+<sub><sup>* All features are dependent on the engine, not all engine have all features.</sub></sub>
```

Comment:
For this part, this may be clearer that this is talking about the engines being supported.

Then as a different set of features, that this project is open source, has automatic installation & config, as well as automatic download of updates as they are supported.